### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by chloe7303**
Sprint 1: Label Management Page
[website](https://github-issues-section.web.app/labels)
[storybook](https://github-issues-storybook.web.app/)

1. Initial Setting
- create react app (CRA)
- react router
- styled component
- typescript

2. OAuth Setting
- sign in/out
- get token

3. Label List function
- list labels
- create a label
- update a label
- delete a label

4. Redux Toolkit (RTK query)
- createApi
- CRUD

5. storybook
- label
- button
- label list item
- dropdown

### CODE REVIEW

**Comment by chloe7303** [comment]
Sprint 2: Issue List Page

Filters function:
1. filter by state (open/closed)
2. filters (your issues/everything assigned to you/everything mentioning you)
3. label
4. assignee
5. sort
- newest/oldest
- most commented/least commented
- recently updated/least recently update

Other function:
- pagination

File: src/components/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+  border-radius: 50%;
+  margin-left: 5px;
+`;
+
+const navlist = ['Pull requests', 'Issues', 'Marketplace', 'Explore'];
+
+function Header() {
+  const [user, setUser] = useState<{} | null>(null);
+  const [token, setToken] = useState('');
+  const [avatarUrl, setAvatarUrl] = useState('');
```
Comment (max80713) [inline src/components/Header.tsx:116]: 多餘的 state，avataUrl 可以從 user object 取得

File: src/pages/IssueList/Dropdown.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+const Dropdown = ({ header, sortList }) => {
+  const [currentFilter, setCurrentFilter] = useState('');
+
+  const handleFilter = (item) => {
+    setCurrentFilter(item.title);
+    item.action();
+  };
+
+  useEffect(() => {
+    if (header === 'Sort by') setCurrentFilter('Newest');
```
Comment (max80713) [inline src/pages/IssueList/Dropdown.tsx:13]: 這樣的邏輯會和 header 的文字還有 sortList 有相依性

File: src/pages/IssueList/IssueItem.tsx [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+                className="font-semibold py-[1px] px-[7px] rounded-[10px] mr-1 text-xs"
+                style={{
+                  backgroundColor: `#${label.color}`,
+                  color: `${lightOrDark(label.color)}`,
+                }}
+              >
+                {label.name}
+              </span>
+            );
+          })}
```
Comment (max80713) [inline src/pages/IssueList/IssueItem.tsx:63]: 這裡的 label 可以拿 Label 元件來共用

**Pull request merged**